src/50_Samples_%26_Templates/Product.html

```diff
@@ -120,10 +127,24 @@
     <body>
     <%>product/header.html%>
 
-        <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
+        <!-- This dropdown allows to select how many items to be added to the shopping cart. Pressing the `ADD TO CART` button should lead to the shopping cart page page. -->
```

Comment:
Can you move the documentation to the form? I'd might make more sense there. Also mention that you're using amp-form.